project_euler/problem_01/sol5.py

```diff
@@ -6,11 +6,25 @@
 '''
 from __future__ import print_function
 try:
-    input = raw_input #python3
+    raw_input          # Python 2
 except NameError:
-    pass               #python 2
+    raw_input = input  # Python 3
```

Comment:
The previous way works as well and has the advantage that it moves the usage to the Python 3 function name but the disadvantages are that it overrides a Python 2 builtin and is not consistent with other files in this repo.

Reply:
I see, thanks for letting me know.